api/db-cdr/generate-cdr/cb-criteria-refactor/debug/make-bq-criteria-tables.sh

```diff
@@ -0,0 +1,6663 @@
+#!/bin/bash
```

Comment:
I think this is Chenchal's final version of this script. Could you verify with `make-bq-criteria-tables.sh` and `make-bq-criteria-tables-10-2021.sh` and `make-bq-criteria-tables_copy.sh` and remove whichever two aren't needed?

Reply:
I would like to keep these until everything is verified working in Circle CI. These are here because one is the script Chenchal based his scripts on. `make-bq-criteria-tables_copy.sh` may be able to be removed if it is the same as `make-bq-criteria-tables.sh`. There were some changes made to `make-bq-criteria-tables.sh` between when Chenchal made his script and when I got the story, so having the one he worked with and the updated one to diff made it a lot easier to tell which pieces of the script needed to be updated. Going forward if there are further changes, having these file will help determine further changes until we can remove `make-bq-criteria-tables.sh`.

Reply:
I have renamed `make-bq-criteria-tables_copy.sh` to `make-bq-criteria-tables_timed.sh`. It looks to be the same as `make-bq-criteria-tables.sh` but with some code for timing how long each section takes.